app/src/main/java/by/paranoidandroid/dailyvisualizer/view/utils/LocationMapManager.java

```diff
@@ -0,0 +1,20 @@
+package by.paranoidandroid.dailyvisualizer.view.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.widget.Toast;
+import by.paranoidandroid.dailyvisualizer.R;
+
+public class LocationMapManager {
+    public static void showLocation(Context context, String lattitude, String longtitude){
+      Uri gmmIntentUri = Uri.parse("geo:" + lattitude + "," + longtitude);
+      Intent mapIntent = new Intent(Intent.ACTION_VIEW, gmmIntentUri);
+      mapIntent.setPackage("com.google.android.apps.maps");
```

Comment:
Maybe is it better to move "com.google.android.apps.maps" to Constants interface?